File: 프로그래머스/월간 코드 챌린지 시즌1/이진 변환 반복하기.py
# 이진 변환 반복하기
def solution(s):
    times,zero = 0,0
    while s != '1':
        times += 1
        zero += len(s)
        s = s.replace("0","")
        zero -= len(s)
        s = bin(len(s))[2:]
    return [times,zero]

File: 프로그래머스/월간 코드 챌린지 시즌1/이진 변환 반복하기.js
// 이진 변환 반복하기
function solution(s){
    let times = 0;
    let zero = 0;
    while(s != '1'){
        times += 1;
        zero += s.length;
        s = s.replace(/0/g,"");
        zero -= s.length
        s = s.length.toString(2);
    }
    return [times,zero]
}